analyzer/codechecker_analyzer/analyzers/clangsa/clang_options.py

```diff
@@ -15,7 +15,7 @@
 LOG = get_logger('analyzer')
 
 
-def get_analyzer_checkers_cmd(cfg_handler, alpha=True, debug=True):
+def get_analyzer_checkers_cmd(cfg_handler, alpha=True, debug=False):
```

Comment:
With this change we will not list `debug` checkers in the output of `CodeChecker checkers` command which is I think is not good. The purpose of this command is to list all the checkers supported by the analyzer. Debug checkers are also checkers and also can be turned on (from this patch by using --saargs only, but it is possible). So from the user perspective it would be good to list these checkers in the output too.

Reply:
End-Users should not use the debug checkers. Debug checkers should only be used by clang static analyzer developers and it is typically not used through codechecker, but rather in the lit test cases and for debugging checker implementations. I think it is enough if we allow users to enable these checkers from teh saargs and describe this as a workaround for edge cases. I will describe this procedure in https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/checker_and_analyzer_configuration.md